public/js/controllers/address_controller.js

```diff
@@ -380,6 +394,32 @@ export default class extends Controller {
     )
   }
 
+  setTablePaginationLinks () {
+    var tablePagesLink = ctrl.tablePaginationParams
+    var txCount = parseInt(ctrl.paginationParams.count)
+    var offset = parseInt(ctrl.paginationParams.offset)
+    var pageSize = parseInt(ctrl.paginationParams.pagesize)
+    var txnType = ctrl.paginationParams.txntype
+    var links = ''
+
+    if (typeof offset !== 'undefined' && offset > 0) {
+      links = `<a href="/address/${this.dcrAddress}?start=${offset - pageSize}&n=${pageSize}&txntype=${txnType}" ` +
+      `class="d-inline-block dcricon-arrow-left m-1 fz20" data-action="click->address#pageNumberLink"></a>` + '\n'
+    }
+
+    links += tablePagesLink.map(d => {
+      if (!d.link) return `<span>${d.str}</span>`
+      return `<a href="${d.link}" class="fs18 pager px-1${d.active ? ' active' : ''}" data-action="click->address#pageNumberLink">${d.str}</a>`
+    }).join('\n')
+
+    if ((txCount - offset) > pageSize) {
+      links += '\n' + `<a href="/address/${this.dcrAddress}?start=${(offset + pageSize)}&n=${pageSize}&txntype=${txnType}" ` +
+      `class="d-inline-block dcricon-arrow-right m-1 fs20" data-action="click->address#pageNumberLink"></a>`
+    }
+
+    ctrl.tablePaginationTarget.innerHTML = links
```

Comment:
Any time we use innerHTML with parameters that might ultimately be set by the user, we should use dompurify. 

```
ctrl.tablePaginationTarget.innerHTML = dompurify.sanitize(links)
```

Reply:
Got it. Its fixed now.